src/applications/vaos/components/ExpressCareReasonField.jsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import classNames from 'classnames';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import { Link } from 'react-router';
+
+export default function ExpressCareReasonField({
+  uiSchema,
+  formData,
+  onChange,
+  errorSchema,
+  formContext,
+  onError,
+}) {
+  const { reason, additionalInformation } = formData;
+  const hasError =
+    formContext.submitted && errorSchema.reason.__errors.length > 0;
+
+  const fieldsetClasses = classNames({
+    'usa-input-error': hasError,
+  });
+
+  return (
+    <>
+      <fieldset className={fieldsetClasses}>
```

Comment:
I'm trying to think through the right way to mark up all the header and field info based on Ryan's designs.

- The radio button inputs all need to have a shared `name` attribute. That makes the standard browser keyboard navigation work correctly.
- I don't think we want this to be a `fieldset`, because `fieldset`s are used when you need to create a group of fields with title text (like the contact info later on). A group of radio buttons has one `name` and is essentially a single form field.
- The text you have in an `h2` in the uiSchema above these fields should probably be in this component as a `label` for the radio button group.

Reply:
I read some differing info, but looks like [W3 recommends](https://www.w3.org/WAI/tutorials/forms/grouping/) grouping radio buttons in a `fieldset` and using a `legend` to describe them.  I moved the `h2` and description text into the component and added a `aria-labelledby` and `role="radiogroup"` to the radio button container.  Do you think this should cover it?

Reply:
Ah, ok, you're right, I misremembered how we have radio buttons set up. The fieldset with a legend label makes sense. I'm not sure if we need a `radiogroup`. The form system by default doesn't, but seems like maybe it should?